src/ApiTemplates/BoredAPICaller.js

```diff
@@ -25,7 +25,7 @@ function BoredAPICaller(props) {
         console.log(error.message);
       }
     })();
-  }, [props.isSubmitPressed]); // Removing the array doesn't work as no element here will
+  }, [props.isSubmitButtonPressed]); // Removing the array doesn't work as no element here will
```

Comment:
Think I would probably destructure the props argument so that you can just say `isSubmitButtonPressed` here